Source/WebKit/Platform/IPC/Decoder.cpp

```diff
@@ -27,58 +27,53 @@
 #include "Decoder.h"
 
 #include "ArgumentCoders.h"
-#include "DataReference.h"
 #include "Logging.h"
 #include "MessageFlags.h"
 #include <stdio.h>
 #include <wtf/StdLibExtras.h>
 
 namespace IPC {
 
-static const uint8_t* copyBuffer(const uint8_t* buffer, size_t bufferSize)
+static DataReference copyBuffer(DataReference buffer)
```

Comment:
I find it a little weird to be storing newly malloc-d memory in a std::span. I'd normally not think of a span as something I need to explicitly free() when I am done.

Maybe that's ok but a little confusing?

Reply:
Okay, I guess that would get rid of the need for `memcpySpan()` below.